rudder-web/src/main/scala/com/normation/rudder/web/components/popup/ModificationValidationPopup.scala

```diff
@@ -312,32 +297,33 @@ class ModificationValidationPopup(
-             else
-                NodeSeq.Empty
-            }
-        </div>
-    val (buttonName, classForButton, titleWorkflow) = (workflowEnabled, action) match {
-      case (false, CreateSolo) => ("Create", "btn-success", NodeSeq.Empty)
-      case (false, CreateAndModRules) => ("Create", "btn-success", NodeSeq.Empty)
-      case (false, Delete) => ("Delete", "btn-danger", NodeSeq.Empty)
-      case (false, Save) => ("Update", "btn-success", NodeSeq.Empty)
-      case (false, Enable) => ("Enable", "btn-primary", NodeSeq.Empty)
-      case (false, Disable) => ("Disable", "btn-primary", NodeSeq.Empty)
-      case (true, CreateSolo) => ("Create" , "btn-success", NodeSeq.Empty)
-      case (true, CreateAndModRules) => ("Open request", "btn-primary", workflowMessage(true))
-      case (true, Delete) => ("Open request", "btn-primary", workflowMessage(false))
-      case (true, Save) => ("Open request", "btn-primary", workflowMessage(false))
-      case (true, Enable) => ("Open request", "btn-primary", workflowMessage(false))
-      case (true, Disable) => ("Open request", "btn-primary", workflowMessage(false))
+    def workflowMessage(directiveCreation: Boolean) = (
+      <h4 class="col-lg-12 col-sm-12 col-xs-12 audit-title">Change Request</h4>
+      <hr class="css-fix"/>
+      <div class="col-lg-12 col-sm-12 col-xs-12 alert alert-info text-center">
+        <span class="glyphicon glyphicon-info-sign"></span>
+        Workflows are enabled in Rudder, your change has to be validated in a Change request
+        {
+          if(directiveCreation)
+            <p>The directive will be directly created, only rule changes have to been validated.</p>
+          else
+            NodeSeq.Empty
+        }
+      </div>
+    )
+    val (buttonName, classForButton, titleWorkflow) = (validationRequired, action) match {
+      case (false, DGModAction.CreateSolo       )   => ("Create"      , "btn-success", NodeSeq.Empty)
+      case (false, DGModAction.CreateAndModRules)   => ("Create"      , "btn-success", NodeSeq.Empty)
+      case (false, DGModAction.Delete           )   => ("Delete"      , "btn-danger", NodeSeq.Empty)
+      case (false, DGModAction.Update             ) => ("Update"      , "btn-success", NodeSeq.Empty)
+      case (false, DGModAction.Enable           )   => ("Enable"      , "btn-primary", NodeSeq.Empty)
+      case (false, DGModAction.Disable          )   => ("Disable"     , "btn-primary", NodeSeq.Empty)
+      case (true , DGModAction.CreateSolo       )   => ("Create"      , "btn-success", NodeSeq.Empty)
+      case (true , DGModAction.CreateAndModRules)   => ("Open request", "btn-primary", workflowMessage(true))
+      case (true , DGModAction.Delete           )   => ("Open request", "btn-primary", workflowMessage(false))
+      case (true , DGModAction.Update             ) => ("Open request", "btn-primary", workflowMessage(false))
```

Comment:
Same :open_mouth:

Reply:
s/Save/Update/g does not correct valign :/